docs/modules/cp-subsystem/pages/configuration.adoc

```diff
@@ -1014,6 +1014,162 @@ config.getCPSubsystemConfig().addSemaphoreConfig(semaphoreConfig);
 ====
 |===
 
+[[cpmap-config]]
+== CPMap Options
+
+Use the following options to configure `CPMap` instances:
+
+[tabs] 
+==== 
+XML:: 
++ 
+--
+[source,xml]
+----
+<hazelcast>
+  <cp-subsystem>
+    <maps>
+      <map>
+        <!-- insert configuration options here -->
+      </map >
+    </maps >
+  </cp-subsystem>
+</hazelcast>
+----
+--
+YAML:: 
++ 
+--
+[source,yaml]
+----
+hazelcast:
+  cp-subsystem:
+    maps:
+      # insert configuration options here
+----
+--
+Java:: 
++ 
+--
+Add configuration options to the link:https://javadoc.io/doc/com.hazelcast/hazelcast/{full-version}/com/hazelcast/config/cp/CPMapConfig.html[`CPMap` object].
+
+[source,java]
+----
+Config config = new Config();
+
+CPMapConfig cpMapConfig = new CPMapConfig(/*insert configuration options here*/);
+
+config.getCPSubsystemConfig().addCPMapConfig(cpMapConfig);
+----
+--
+====
+
+[cols="1a,1a,1m,2a",options="header"]
+|===
+|Option|Description|Default|Example
+
+
+|[[cpmap-name]]`name`
+|Name of the CPMap.
+a|' ' (empty)
+
+|
+[tabs] 
+==== 
+XML:: 
++ 
+--
+[source,xml]
+----
+<hazelcast>
+  <cp-subsystem>
+    <maps>
+      <map>
+          <name>map1</name>
+      </map>
+    </maps>
+  </cp-subsystem>
+</hazelcast>
+----
+--
+YAML:: 
++ 
+--
+[source,yaml]
+----
+hazelcast:
+  cp-subsystem:
+    maps:
+      map1:
+----
+--
+Java:: 
++ 
+--
+[source,java]
+----
+Config config = new Config();
+
+CPMapConfig cpMapConfig = new CPMapConfig("map1");
+
+config.getCPSubsystemConfig().addCPMapConfig(cpMapConfig);
+----
+--
+====
+
+|[[max-size-mb]]`max-size-mb`
+|Maximum size in MB the key-value data is permitted to total.
```

Comment:
Maximum permitted size in MB for the totality of key-value data

Reply:
Done.